platforms/core-configuration/configuration-cache/src/smokeIdeTest/groovy/org/gradle/configurationcache/AbstractIdeSyncSmokeTest.groovy

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.configurationcache
+
+import com.google.common.collect.ImmutableList
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.internal.UncheckedException
+import org.gradle.internal.jvm.Jvm
+import org.gradle.profiler.BuildAction
+import org.gradle.profiler.BuildMutator
+import org.gradle.profiler.GradleBuildConfiguration
+import org.gradle.profiler.InvocationSettings
+import org.gradle.profiler.Logging
+import org.gradle.profiler.Profiler
+import org.gradle.profiler.ScenarioContext
+import org.gradle.profiler.gradle.DaemonControl
+import org.gradle.profiler.gradle.GradleBuildInvoker
+import org.gradle.profiler.gradle.GradleScenarioDefinition
+import org.gradle.profiler.gradle.GradleScenarioInvoker
+import org.gradle.profiler.instrument.PidInstrumentation
+import org.gradle.profiler.studio.AndroidStudioSyncAction
+import org.gradle.profiler.studio.invoker.StudioBuildInvocationResult
+import org.gradle.profiler.studio.invoker.StudioGradleScenarioDefinition
+import org.gradle.profiler.studio.invoker.StudioGradleScenarioInvoker
+import org.gradle.profiler.studio.tools.StudioFinder
+
+import java.util.function.Consumer
+
+abstract class AbstractIdeSyncSmokeTest extends AbstractIntegrationSpec {
+
+    protected IdeSyncFixture fixture = new IdeSyncFixture(testDirectory)
+
+    protected StudioBuildInvocationResult syncResult
+
+    protected void androidStudioSync() {
+        assert System.getenv("ANDROID_HOME") != null
```

Comment:
Performance tests are using `ANDROID_SDK_ROOT`, which seems like to be deprecated https://developer.android.com/tools/variables#envar in favor of `ANDROID_HOME`